cmake/OpenVDBHoudiniSetup.cmake

```diff
@@ -302,9 +302,6 @@ endif()
 # Boost - currently must be provided as VDB is not fully configured to
 # use Houdini's namespaced hboost
 
-unset(_HOUDINI_INCLUDE_DIR)
-unset(_HOUDINI_LIB_DIR)
```

Comment:
Perhaps lets rename these to `HOUDINI_INCLUDE_DIR` and `HOUDINI_LIB_DIR` and add them to the comment at the top of this file ad advertised variables which can be used